addons/website_slides_survey/models/slide_slide.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class SlidePartnerRelation(models.Model):
+    _inherit = 'slide.slide.partner'
+
+    user_input_ids = fields.One2many('survey.user_input', 'slide_partner_id', 'Certification attempts')
+    survey_quizz_passed = fields.Boolean('Certification Quizz Passed', compute='_compute_survey_quizz_passed')
+
+    @api.depends('partner_id', 'user_input_ids.quizz_passed')
+    def _compute_survey_quizz_passed(self):
+        for record in self:
```

Comment:
Are you sure you won't run into access rights issues ? Only survey officers / managers can read user inputs. I think you will have to do a search / read_group using a sudo, directly on user_input field. That way we don't need the o2m .

Reply:
Changed it to an explicit "sudo" compute as discussed.